great_expectations/data_context/types/base.py

```diff
@@ -410,12 +414,13 @@ def validate_schema(self, data, **kwargs):
         ):
             raise ge_exceptions.InvalidConfigError(
                 f"""Your current configuration uses one or more keys in a data connector, that are required only by a
-filesystem type of the data connector (your data conntector is "{data['class_name']}").  Please update your
+filesystem type of the data connector (your data connector is "{data['class_name']}").  Please update your
 configuration to continue.
                 """
             )
         if (
-            "bucket" in data
+            "credentials" in data
+            or "bucket" in data
```

Comment:
Note that `credentials` are only ever used in S3 (where `DataConnector`s are concerned).

Is this appropriate? None of the other connectors have connection objects like `boto3`. If other connectors utilize these newfound `credentials`, I will have to fix up `validate_schema()`